ts/main.ts

```diff
@@ -302,6 +302,11 @@ async function fetchData(usrInput: string): Promise<void> {
       $selectedSq.classList.remove('hover:bg-yellow-100');
       clearSelectedSqResetModal($selectedSq);
     }
+    if (completedSquares === gridSize * gridSize && $statsModal) {
+      setStatsText();
+      $statsModal.show();
+      console.log('fired');
+    }
   } catch (error) {
     console.log('Error: ', error);
   }
```

Comment:
You should change this one to `console.error` instead of `console.log` too. They will look similarly here but in a different type of application a `console.log` may still print to the browser console and you don't want the users to get the error if possible.